### ISSUE TITLE
How to discard/stash commits?

### ISSUE THREAD

**Issue by lmumar** [SquaredTiki/Dringend-Issues#80]
Hello again,

There are instances that you don't want to push your commits to github, if I had a terminal I would just do git checkout <target file>, in dringend how to do that? The problem is pulling changes from your repo will cause an error. There's also no way to stash currently what you are doing.

Thanks

**Comment by SquaredTiki** [comment]
Are you wanting to revert to a previous commit then and then pull from the remote repo? Unfortunately it's not possible to stash changes at present, I'll count that as a feature request though and shall certainly look into it.

Josh

**Comment by lmumar** [comment]
There are two scenarios:
- I want to pull changes from other developer without commiting my changes first. For that I would have to stash my changes, pull from the repo and stash apply my changes on top
- I want to discard all my changes and just revert to the previous commit. 

Thanks

**Comment by SquaredTiki** [comment]
Thanks very much for describing the situations. I'll look into implementing stashing and reverting in a future update. Apologies they are not available at present.

Josh (📱)

**Comment by dgileadi** [comment]
I'd like to add my vote to this request. Right now I have two files changed on my iPad, one of which I changed and one of which is an .xcscheme file and so which I can't change. I can't pull, since I have pending changes, and I can't revert, so I'm stuck.

**Comment by SquaredTiki** [comment]
@dgileadi Thanks for adding your vote for the request, I'll make this a priority for the next update. Do you not wish to commit the changes and would rather discard them and revert to the last commit?

**Comment by dgileadi** [comment]
Exactly, I'd like the ability to discard uncommitted changes and revert them to the last commit.

**Comment by SquaredTiki** [comment]
@dgileadi Got it, will look into this for the next update.
